source/features/raw-view-link.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const childList = {childList: true};
+
+const createRawUrl = (href: string): string => {
+	const url = href.split('/');
+	url[3] = 'raw'; // Replaces 'blob'
+	return url.join('/');
+};
+
+const observer = new MutationObserver(([{addedNodes}]) => {
+	for (const node of addedNodes) {
+		const element = node as Element;
+		if (element.tagName === 'DIV') {
+			if (element.className === 'js-diff-progressive-container') {
+				observer.observe(element, childList);
+			} else {
+				addRawBtn(element);
+			}
+		}
+	}
+});
+
+function addRawBtn(node: Element) {
+	for (const link of select.all('.file-header .file-actions > div.BtnGroup [href*=\'/blob/\']', node)) {
+		link.after(
+			<a href={createRawUrl(link.getAttribute('href'))} class="btn btn-sm BtnGroup-item">
+				Raw
+			</a>
+		);
+	}
+}
+
+function init() {
+	addRawBtn(select('.js-diff-progressive-container:first-child'));
+	observer.observe(select('.js-diff-progressive-container:last-child'), childList);
```

Comment:
It is the last child. Please see my comment above https://github.com/sindresorhus/refined-github/pull/1924#issuecomment-482838249

Reply:
> It is the last child

In the HTML you posted I see two `:last-child`, and I think `querySelector` will match the first one (level 2), not the one you expect (level 3)

See https://github.com/sindresorhus/refined-github/pull/1932/commits/a6a6e15d1d0300789b4f548dffb7d5777ad10bc8 for an alternative implementation that avoids this altogether

Also do you know a PR that has more than one level? Because in the biggest PR I can think of (https://github.com/sindresorhus/refined-github/pull/1783) I'm seeing only one deferred level, _not_ nested:


```html
<div class="js-diff-progressive-container">already here</div>
<div class="js-diff-progressive-container">(content deferred)</div>
```

Reply:
I like your approach on the other PR 👍 
In the HTML I've posted I was specifically talking about the container that gets updated -- which is the `:last-child` in the beginning (the one with deferred content). I.e. the HTML will, in the end, have a format of:
```
<div class="js-diff-progressive-container">already here</div>
<div class="js-diff-progressive-container">
  (nested containers)
</div>
```